src/libDirectoryService/PoWProcessing.cpp

```diff
@@ -151,6 +151,12 @@ bool DirectoryService::ProcessPoWSubmission(
     return true;
   }
 
+  if (m_powSubmissionWindowExpired) {
```

Comment:
Should we put this within UPDATE_TARGET_DS_NUM?

Reply:
We don’t need right. We can deploy one node ( non-leader) at a time.

Reply:
ok, agree